app/helpers/application_helper.rb

```diff
@@ -9,4 +11,13 @@ def alert_class_for(flash_type)
     }
     types[flash_type.to_sym]
   end
+
+  def all_months_select_options(first_month, last_month, end_of_month_flg = false)
+    all_months(first_month, last_month, end_of_month_flg).map { |month| [month.strftime('%Y年%m月'), month] }
+  end
+
+  def all_months(first_month, last_month, end_of_month_flg = false)
+    months = (Month.new(first_month.year, first_month.month)..Month.new(last_month.year, last_month.month))
+    end_of_month_flg ? months.map(&:end_date) : months.map(&:start_date)
```

Comment:
__[reek]__
* [ControlParameter] is controlled by argument 'end_of_month_flg' [:ref](https://github.com/troessner/reek/blob/master/docs/Control-Parameter.md)

<!-- Comment by [SideCI](https://sideci.com) -->